src/main/java/Person.java

```diff
@@ -45,7 +46,7 @@ public String getFullName(){
         if(lastName.isEmpty()){
             lastName = "";
         }
-        String a = firstName+' '+lastName;
+        String a = firstName+""+lastName;
         return a;
```

Comment:
1. Leave the space between firstName and lastName
2. You can use few return statements to return only firstName or only lastName in appropriate cases as unit test fail du to the following: Error details
Method call getFullName() returned a value of "John ", but "John" was expected. expected:<John[]> but was:<John[ ]>
You don't need to set firstName = "". Just do not return it